src/maker.rs

```diff
@@ -88,44 +82,38 @@ impl Maker {
         self.dai_balance = balance;
     }
 
-    pub fn track_failed_balance_update(&mut self, error: anyhow::Error) {}
-
-    fn rate_is_up_to_date(&self) -> bool {
-        let current_timestamp = Utc::now().timestamp();
-        let mid_market_rate_timestamp = self.mid_market_rate.timestamp.timestamp();
-        let acceptable_timestamp = (current_timestamp - self.max_rate_ticker_interval_secs);
-
-        mid_market_rate_timestamp >= acceptable_timestamp
+    pub fn track_failed_rate_update(&mut self) {
+        self.mid_market_rate = None;
     }
 
+    pub fn track_failed_balance_update(&mut self, error: anyhow::Error) {}
```

Comment:
:wrench: You can remove it if it is unused (for now).

Reply:
That has been in there for long already :D 
Removed!